content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md

```diff
@@ -62,37 +61,53 @@ Alternatively, customize this CloudFormation template and install it from the AW
 
 ## Send AWS logs to your Kinesis stream
 
-1. Check the `Subscriptions` column on the [log groups index page][1] to see current subscriptions to your relevant log groups. Add the new Kinesis stream as a subscriber.    
-   * **Note**: Each CloudWatch Log group can only have two subscriptions. If you have more than two sources you want to subscribe to, you can subscribe to the new Kinesis stream after completing this setup.
-2. Subscribe your new Kinesis stream to the CloudWatch log groups you want to ingest into Datadog. Refer to [this CloudWatch Logs documentation section][2] (step 3 to 6) to:  
-   - Use the `aws iam create-role` command to create the IAM role that gives CloudWatch Logs permission to put logs data into the Kinesis stream.
-   - Create a permissions policy allowing the `firehose:PutRecord` `firehose:PutRecordBatch`, `kinesis:PutRecord`, and `kinesis:PutRecords` actions.
+Subscribe your new Kinesis stream to the CloudWatch log groups you want to ingest into Datadog. You can check the **Subscriptions** column on the [log groups index page][1] to see current subscriptions to your log groups. Subscriptions can be created through the AWS console or API with the specifications below.  
+   **Note**: Each CloudWatch Log group can only have two subscriptions.
+
+### Create an IAM Role and Policy
+
+Create an IAM role and permissions policy to enable CloudWatch Logs to put data into your Kinesis stream. 
    - Ensure that `logs.amazonaws.com` or `logs.<region>.amazonaws.com` is configured as the service principal in the role's **Trust relationships**.
-   - Attach the permissions policy to your newly created IAM role using the `aws iam put-role-policy` command.
-   - Use the `aws logs put-subscription-filter` command to subscribe your Kinesis stream to each CloudWatch log group you want to ingest into Datadog.  
+   - Ensure that the role's attached permissions policy allows the `firehose:PutRecord` `firehose:PutRecordBatch`, `kinesis:PutRecord`, and `kinesis:PutRecords` actions.
+
+Use the [Subscription filters with Kinesis][2] example (steps 3 to 6) for an example of setting this up with the AWS CLI.
+
+### Create a subscription filter
+
+The following example creates a subscription filter through the AWS CLI:
 
-   Example of subscription filter:
     ```
     aws logs put-subscription-filter \
-        --log-group-name "MYLOGGROUPNAME" \
-        --filter-name "MyFilterName" \
+        --log-group-name "<MYLOGGROUPNAME>" \
+        --filter-name "<MyFilterName>" \
         --filter-pattern "" \
-        --destination-arn "DESTINATIONARN (data stream or delivery stream)" \
-        --role-arn "MYROLEARN"
+        --destination-arn "<DESTINATIONARN> (data stream or delivery stream)" \
+        --role-arn "<MYROLEARN>"
     ```
```

Comment:
This code block isn't formatting quite correctly. I think you want zero spaces before the backticks on lines 79 and 86

Reply:
Thanks for taking a look, this was added in [2a9f24a](https://github.com/DataDog/documentation/pull/16337/commits/2a9f24a424b1e8de8428d367afadbc528fbc78e4), and the code block seems to be rendering in the current preview.